packages/itinerary-body/src/types.ts

```diff
@@ -199,6 +199,8 @@ interface ItineraryBodySharedProps {
   setLegDiagram: (leg: Leg) => void;
   /** Fired when a user clicks on a view trip button of a transit leg */
   setViewedTrip: SetViewedTripFunction;
+  /** Whether to prefix non transit legs with 'About' text and show the '~' symbol in the trip details panel */
```

Comment:
This prop does not affect the trip details panel, so remove the part of the comment that is about that.